core/src/com/unciv/models/ruleset/tile/TileImprovement.kt

```diff
@@ -32,7 +32,13 @@ class TileImprovement : NamedStats() {
 
     fun getDescription(ruleset: Ruleset): String {
         val stringBuilder = StringBuilder()
-        if (this.clone().toString().isNotEmpty()) stringBuilder.appendln(this.clone().toString())
+        val statsDesc = this.clone().toString()
+        if (statsDesc.isNotEmpty()) stringBuilder.appendln(statsDesc)
+        if (improvingTech != null && improvingTechStats != null) {
+            val improveStatsDesc = improvingTechStats.toString()
+            if (improveStatsDesc.isNotEmpty()) stringBuilder.appendln("$improveStatsDesc ${"with".tr()} ${improvingTech!!.tr()}")
```

Comment:
Please change this format to "[$a] with [$b]", so that it can be translated to all languages and is clearer in translation

Reply:
Where did my comment from yesterday go? Anyway, I decided against significant leading and trailing blanks in a translation key. Having " with  = " work but " with = " break?

Reply:
I mean, technically it could work, but since this is already supposed to be part of a specific sentence, it's much clearer for the translators how to translate it if they have the context of the sentence

Reply:
(facepalm) knot in brain. _Of course_ you're right! But - you see the "from unknown repository" tag? ...